build/Linux+BSD/musescore.xml.in

```diff
@@ -103,7 +103,8 @@
     <mime-type type="application/vnd.recordare.musicxml">
         <!-- See "Container file" at http://www.musicxml.com/for-developers/musicxml-dtd/ -->
         <comment>MusicXML compressed score</comment>
-        <glob pattern="*.mxl"/>
+        <glob pattern="*.mxl"/><!-- preferred extension must go first -->
+        <glob pattern="*.mxml" weight="40"/><!-- reduced weight (default 50) to avoid matching ordinary XML files -->
```

Comment:
This line should be:
```xml
<glob pattern="*.mxml" weight="40"/><!-- reduced weight (default 50) to avoid matching Macromedia XML files -->
```
Or if we don't care about Macromedia XML files:
```xml
<glob pattern="*.mxml">
```
There is no requirement to reduce the weight. We only did it for `.xml` so that the operating system wouldn't mistake random XML files for MusicXML files just because they have a `.xml` extension.

Reducing the weight of `*.mxml` has no effect on how the OS handles `*.mxl`.

Reply:
I see, will amend, tomorrow

Reply:
done